source/create.js

```diff
@@ -51,18 +53,18 @@ const create = defaults => {
 		return defaults.handler(url, options, next);
 	};
 
+	got.hooks = defaults.options.hooks = { // eslint-disable-line no-multi-assign
```

Comment:
The ESLint rule is there for a reason ;)

Reply:
I don't understand what's wrong with that :P I'll change it :)

Reply:
In this case, nothing major, but very often it creates harder to read code.